org.kde.skanpage.json

```diff
@@ -0,0 +1,251 @@
+{
+    "id": "org.kde.skanpage",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-23.08",
+    "sdk": "org.kde.Sdk",
+    "rename-icon": "skanpage",
+    "command": "skanpage",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=all",
+        "--share=network",
+        "--system-talk-name=org.freedesktop.Avahi"
+    ],
```

Comment:
It's missing any cleanups. There are lots of stuff to clean up from the final build.

Reply:
```
- "*.a"
- "*.la"
- /include
- /lib/cmake
- /lib/pkgconfig
- /mkspecs
- /share/doc
- /share/man
- /share/qlogging-categories5
```

There's an empty `/app/var/lock` folder being created.